AnimeshKumarRai/task4.html

```diff
@@ -0,0 +1,49 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>task4</title>
+    <link rel="stylesheet" href="/Web/Task/web-development-task1/assets/css/navbar.css">
+    <style>
+        body {
+            margin: 0;
+            font-family: Arial, Helvetica, sans-serif;
+        }
+        h1{
+            justify-content: center;
+            align-items: center;
+            display: flex;
+        }
+    </style>
+</head>
+<body>
+    <div class="navbar">
+        <a href="#home">Home</a>
+        <a href="#about">About</a>
+        <a href="#actions">Actions</a>
+    </div>
+    <pre>
+        <div id="home" style="padding-top:5px">
+            <h1> Hello World</h1>
+        </div>
+
+        <div style="line-height:60em">.</div>
```

Comment:
Great way to space the sections, here is a suggestion (just for trying it out).
Use `height: 100vh` in style for the home div, which means 100% vertical height.
It will allow you to set height according to screen sizes.